app/src/main/java/com/concordium/wallet/core/config/EnvironmentConfiguration.kt

```diff
@@ -0,0 +1,8 @@
+package com.concordium.wallet.core.config
+
+object EnvironmentConfiguration {
+    const val baseUrl = "https://wallet-proxy.mainnet.concordium.software"
+    const val maxTimeout = 10L
+
+    const val SHARED_PREFERENCES_FILE_NAME = "concordiumPref"
+}
```

Comment:
Please set up your editor to automatically end the file with a newline.

Reply:
I never understood this approach. could I ask what is significant in the extra line or lack of it at the end of the file?